.eslintrc.js

```diff
@@ -6,15 +6,12 @@ module.exports = {
         '@jitsi/eslint-config'
     ],
     rules: {
-        "max-len": ["error", { "code": 120 }], // Limit line length to 120 characters
-        "key-spacing": ["error", { "beforeColon": false, "afterColon": true }], // Enforce spacing in object keys
-        "func-call-spacing": ["error", "never"], // No space between function name and parentheses
-        "no-console": ["warn", { "allow": ["warn", "error"] }],
-        "no-new": ["error"],
         "@typescript-eslint/no-import-type-only": ["error"],
         "logical-assignment-operators": ["error", "always"],
         "no-unused-private-class-members": ["error"],
         "no-useless-backreference": ["error"],
-        "require-await": ["error"] 
+        "require-await": ["error"],
+        "no-floating-promises": ["error"],
```

Comment:
This one likely generates a lot of false positives, I wouldn't set it just yet.

However, setting it locally and fixing legit cases would be good!